packaging/chocolatey/crc/README.md

```diff
@@ -0,0 +1,93 @@
+﻿# Chocolatey package for CRC (non GUI)
+
+## Pre-requisite
+
+The host machine needs to have [Chocolatey](https://chocolatey.org/) installed and should be able to build `crc`. Follow [chocolatey install guide](https://chocolatey.org/install) to install it.
+
+## Steps to build the package
+
+1. Build `crc` executable for windows:
+```
+PS> make out/windows-amd64/crc.exe
+```
+2. Navigate to the chocolatey packaging folder (current folder)
+```
+PS> pwd
+
+Path
+----
+C:\Users\anath\redhat\crc\packaging\chocolatey\crc
+```
+3. Run the script `update.ps1` to update the package spec files with latest version numbers, the script fetches the latest tag from the Github repository and uses that as the latest version
+

```

Comment:
Step 1. builds crc.exe from the current local git HEAD, why do we use the latest remote tag to version it? Shouldn't the version be derived from `git describe HEAD` instead?
If I do `git checkout v2.9.0; make out/windows-amd64/crc.exe` and then run `update.ps1`, it feels wrong to version this as `2.12.0`

Reply:
this is handled now after addressing https://github.com/crc-org/crc/pull/3444#discussion_r1061604661 since everything is incorporated in the `Makefile`